doc/sphinx/language/core/sections.rst

```diff
@@ -131,6 +131,90 @@ usable outside the section as shown in this :ref:`example <section_local_declara
    Notice the difference between the value of :g:`x'` and :g:`x''` inside section
    :g:`s1` and outside.
 
+.. _visibility-attributes-sections:
+
+Summary of locality attributes in a section
+-------------------------------------------
+
+The following table sums up the locality of vernacular commands in sections, when
+outside the section where it was entered.
```

Comment:
```suggestion
Scope of locality attributes in a section
-----------------------------------------

This table summarizes the scope of the effects of vernacular commands outside
the section containing the command.
```

Better?  If you with this, you may want to tweak the wording used in table entries.  WDYT?

Reply:
I'm not sure about this one. Scope would be the corresponding term in other programming languages, but in Coq it is really tight to notations and I don't want to create more confusion.

Reply:
Many of the terms we use in Coq doc have multiple meanings depending on the context.  For example "scope" is already used in the programming language sense in the modules chapter for describing the locality attributes:

![image](https://github.com/coq/coq/assets/1253341/25ca18fb-100e-41be-869b-999cb5ec497c)

When necessary, we try to be more explicit in the doc, e.g. there are references to "notation scope", which is in the glossary BTW.

Reply:
I have not changed the subtitle, but mentioned the scope in the explanation.